### PR TITLE
Calculator with interactive UI

### PR DESCRIPTION
**Pull request by abhishek310001**
# Description:
This PR introduces the Calculator Project from the Odin Project's Foundation course. It allows users to perform basic mathematical operations such as addition, subtraction, multiplication and division.
But, before moving on we have to press the **Equal** button to display the results on the screen.

## Feature:

- Decimal number operations can also be done with this project
- Floating numbers are accurate up to three decimal digits
- Also added a mouse hover effect on buttons using CSS only
- Result screen can show up to 10 digits

## UI Screenshot:
Below is the screenshot of the UI for this PR

calculator-project_

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+body{
+    display: flex;
+    align-items:center;
+    flex-direction: column;
+    justify-content: center;
+    width: 100vw;
+    height: 100vh;
+}
+h1{
```
Comment (devanuj97) [inline style.css:10]: add space here and other place as well

File: script.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+let prevValue;
+let currValue = '';
+let result = 0;
+const display = document.querySelector('.display');
+const numberBtns = document.querySelectorAll('.btn.numbers');
+const operatorBtns = document.querySelectorAll('.btn.operators');
+const equalBtn = document.querySelector('.btn.equal');
+const clearBtn = document.querySelector('.btn.clear');
+const decimalBtn = document.querySelector('.btn.decimal');
+
```
Comment (devanuj97) [inline script.js]: declare the variable at the top

**Review by devanuj97 — APPROVED** [review]

**Comment by devanuj97** [comment]
@abhishek310001 have you covered `point 7` of the odin project 

```
Gotchas: watch out for and fix these bugs if they show up in your code:
Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, 12 + 7 - 5 * 3 = should yield 42. An example of the behavior we’re looking for would be [this student solution](https://mrbuddh4.github.io/calculator/).
Your calculator should not evaluate more than a single pair of numbers at a time. Example: you press a number button (12), followed by an operator button (+), a second number button (7), and finally a second operator button (-). Your calculator should then do the following: first, evaluate the first pair of numbers (12 + 7), second, display the result of that calculation (19), and finally, use that result (19) as the first number in your new calculation, along with the next operator (-).
You should round answers with long decimals so that they don’t overflow the screen.
Pressing = before entering all of the numbers or an operator could cause problems!
Pressing “clear” should wipe out any existing data.. make sure the user is really starting fresh after pressing “clear”
Display a snarky error message if the user tries to divide by 0… and don’t let it crash your calculator!
```
Screenshot 2023-04-13 at 5 48 57 PM

**Comment by abhishek310001** [comment]
> @abhishek310001 have you covered `point 7` of the odin project
> 
> ```
> Gotchas: watch out for and fix these bugs if they show up in your code:
> Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, 12 + 7 - 5 * 3 = should yield 42. An example of the behavior we’re looking for would be [this student solution](https://mrbuddh4.github.io/calculator/).
> Your calculator should not evaluate more than a single pair of numbers at a time. Example: you press a number button (12), followed by an operator button (+), a second number button (7), and finally a second operator button (-). Your calculator should then do the following: first, evaluate the first pair of numbers (12 + 7), second, display the result of that calculation (19), and finally, use that result (19) as the first number in your new calculation, along with the next operator (-).
> You should round answers with long decimals so that they don’t overflow the screen.
> Pressing = before entering all of the numbers or an operator could cause problems!
> Pressing “clear” should wipe out any existing data.. make sure the user is really starting fresh after pressing “clear”
> Display a snarky error message if the user tries to divide by 0… and don’t let it crash your calculator!
> ```
> 
> Screenshot 2023-04-13 at 5 48 57 PM

i thought i covered this one but i forgot to check its testcases whether its working or not
Give me a moment to figure this out and i'll let you know as soon as its solved

**Review by devanuj97 — APPROVED** [review]

**Pull request merged**